src/app/dataValidator/rules/topstories.js

```diff
@@ -0,0 +1,55 @@
+import deepClone from '../../helpers/json/deepClone';
+import deepGet from '../../helpers/json/deepGet';
+
+const squashKeys = [
+  'responsive-top-stories',
+  'top-story',
+  'secondary-top-story',
+  'other-top-stories',
+];
```

Comment:
Are we aware that there are other groups we will need to flatten in the future? The current 4 services follow the "new" content group structure:
```
 'responsive-top-stories',
  'top-story',
  'secondary-top-story',
  'other-top-stories'
```

Whereas the content group structure for the older service is:
```

```